### ISSUE TITLE
Clean up PHP references in EEX templates

### ISSUE THREAD

**Issue by gamebox** [erlef/website#17]
There are a few straggling PHP references sprinkled around the templates, mostly of the form:

```html
<!-- ?php the_title( '<h1 class="entry-title h2">', '</h1>' ); ? -->
```

Let's get the definition of this php function and replace the calls with the appropriate `eex` or static html as appropriate

**Comment by starbelly** [comment]
Haha yeah, I thought about us leaving this as a historical joke... but yeah, needs a 🔪 taken to it.

**Comment by gamebox** [comment]
I'm wondering if `the_title(...)` provided some sort of string content?  It's often contained in an empty `<header>` tag.  Some of the pages that have it have no `<h1>` or `<h2>` tags at all.  There is also still some `jetpack` specific classes attached

**Comment by starbelly** [comment]
Not sure. Wouldn't really worry about that, rather just making sure we have all the content in sync (doesn't mean looking exactly the same). Some of that is of course in flux at the moment and will remain so until @desmondmonster can chime in (he's traveling right now).

**Comment by gamebox** [comment]
Sounds good

**Comment by desmondmonster** [comment]
@gamebox I agree with @starbelly that we can simply excise the php code if the final layout looks okay.  At "some point" in the future we'll do a more thorough site refresh and can make sure all the h1s and h2s are what/where they should be.  In the meantime, if it looks good, 🚢

**Comment by gamebox** [comment]
Sounds good.

**Comment by starbelly** [comment]
Closed via #15
